src/Bicep.Cli/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
```

Comment:
Should we just gitignore this file?

Reply:
Yeah, I think so. I'll make the change in this PR.

---
In reply to: [582065438](https://github.com/Azure/bicep/pull/1521#discussion_r582065438) [](ancestors = 582065438)

Reply:
Coming in next commit

---
In reply to: [582209011](https://github.com/Azure/bicep/pull/1521#discussion_r582209011) [](ancestors = 582209011,582065438)

Reply:
Done.